src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config

```diff
@@ -2,5 +2,8 @@
 <packages>
   <package id="log4net" version="2.0.5" targetFramework="net45" />
   <package id="Microsoft.ApplicationInsights" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
```

Comment:
I don't think I do but I was putting back all of the references which had existed in develop prior to the merges I made that took away some references that shouldn't have been removed.
I can try removing these again and see if the new build still passes.